doc/sphinx/changes.rst

```diff
@@ -6,6 +6,492 @@ Recent changes
+- **Removed:** Legacy commands ``AddPath``, ``AddRecPath``, and ``DelPath``
+  which were undocumented, broken variants of :cmd:`Add LoadPath`,
+  :cmd:`Add Rec LoadPath`, and :cmd:`Remove LoadPath`
+  (`#11187 <https://github.com/coq/coq/pull/11187>`_,
+  by Maxime Dénès and Théo Zimmermann).
+
+**Tools**
+
+  .. _811vos:
+
+- **Added:**
+  `coqc` now provides the ability to generate compiled interfaces.
+  Use `coqc -vos foo.v` to skip all opaque proofs during the
+  compilation of `foo.v`, and output a file called `foo.vos`.
+  This feature is experimental. It enables working on a Coq file without the need to
+  first compile the proofs contained in its dependencies
+  (`#8642 <https://github.com/coq/coq/pull/8642>`_ by Arthur Charguéraud, review by
+  Maxime Dénès and Emilio Gallego).
+- **Added:**
+  Command-line options `-require-import`, `-require-export`,
+  `-require-import-from` and `-require-export-from`, as well as their
+  shorthand, `-ri`, `-re`, `-refrom` and -`rifrom`. Deprecate
+  confusing command line option `-require`
+  (`#10245 <https://github.com/coq/coq/pull/10245>`_
+  by Hugo Herbelin, review by Emilio Gallego).
+- **Changed:**
+  Renamed `VDFILE` from `.coqdeps.d` to `.<CoqMakefile>.d` in the `coq_makefile`
+  utility, where `<CoqMakefile>` is the name of the output file given by the
+  `-o` option. In this way two generated makefiles can coexist in the same
+  directory.
+  (`#10947 <https://github.com/coq/coq/pull/10947>`_, by Kazuhiko Sakaguchi).
+- **Fixed:** ``coq_makefile`` now supports environment variable ``COQBIN`` with
+  no ending ``/`` character (`#11068
+  <https://github.com/coq/coq/pull/11068>`_, by Gaëtan Gilbert).
+
+**Standard library**
+
+- **Changed:**
+  Moved the :tacn:`auto` hints of the `OrderedType` module into a new `ordered_type`
+  database
+  (`#9772 <https://github.com/coq/coq/pull/9772>`_,
+  by Vincent Laporte).
+- **Removed:**
+  Deprecated modules `Coq.ZArith.Zlogarithm` and `Coq.ZArith.Zsqrt_compat`
+  (`#9881 <https://github.com/coq/coq/pull/9811>`_,
+  by Vincent Laporte).
+
+  .. _811Reals:
+
+- **Added:**
+  Module `Reals.ConstructiveCauchyReals` defines constructive real numbers
+  by Cauchy sequences of rational numbers
+  (`#10445 <https://github.com/coq/coq/pull/10445>`_, by Vincent Semeria,
+  with the help and review of Guillaume Melquiond and Bas Spitters).
+
+- **Changed:** Classical real numbers are now defined
+  as a quotient of these constructive real numbers, which significantly reduces
+  the number of axioms needed (see `Reals.Rdefinitions` and `Reals.Raxioms`),
+  while preserving backward compatibility.
+
+  Futhermore, the new axioms for classical real numbers include the limited
+  principle of omniscience (`sig_forall_dec`), which is a logical principle
+  instead of an *ad hoc* property of the real numbers.
+
+  (`#10445 <https://github.com/coq/coq/pull/10445>`_, by Vincent Semeria,
+  with the help and review of Guillaume Melquiond and Bas Spitters)
```

Comment:
This whole bullet-point is made obsolete by the next one, so this should probably simply be removed. The part "which significantly reduces the number of axioms needed (see `Reals.Rdefinitions` and `Reals.Raxioms`), while preserving backward compatibility" is the interesting part that could be copy-pasted to the next bullet point.